src/components/Home.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
-import { Grid, CssBaseline, useMediaQuery } from '@material-ui/core';
+import { Grid, CssBaseline, useMediaQuery } from '@mui/material';
 import Calendar from './Calendar/ScheduleCalendar';
 import Bar from './AppBar/CustomAppBar';
 import DesktopTabs from './RightPane/RightPaneRoot';
 import NotificationSnackbar from './AppBar/NotificationSnackbar';
-import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import MobileHome from './MobileHome';
-import DateFnsUtils from '@date-io/date-fns';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
```

Comment:
Is this being used anywhere? Also, on a higher-level note why do we need to wrap the entire site in this provider component and why are we replacing it?

Reply:
I asked in the discord. I am not sure we need this anymore.